Formula/pandoc-crossref.rb

```diff
@@ -3,7 +3,7 @@ class PandocCrossref < Formula
   homepage "https://github.com/lierdakil/pandoc-crossref"
   url "https://hackage.haskell.org/package/pandoc-crossref-0.3.7.0/pandoc-crossref-0.3.7.0.tar.gz"
   sha256 "467d2470fa66885179284e54d8757ae1a9b442b6ff8af33bd177870f285b85d7"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
https://github.com/lierdakil/pandoc-crossref#license

> This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.